main/ertp/ERTP-Guide/PursesAndPayments.md

```diff
@@ -0,0 +1,269 @@
+# Purses and Payments
+
+You can store `amount` objects in two kinds of objects:
+- **[`purse`](https://agoric.com/documentation/glossary/#purse)**: Hold
+  an `amount` of same branded digital assets until you withdraw part or
+  all of them into a `payment` for use.
+- **[`payment`](https://agoric.com/documentation/glossary/#purse)**:
+  Hold an `amount` of digital assets to send to another party. Usually either
+  a freshly created `amount` or one taken from a `purse`
+
+Assets in purses and payments do not have to be currency-like, but can
+be anything you can construct a digital rights `amount` object for;
+swords to use in a game, rights to use a particular contract, theater
+tickets, etc.
+
+Each non-empty `purse` and `payment` object contains exactly one
+`amount` object. While you may deposit or withdraw an `amount` to a
```

Comment:
No, we really need to correct this. There is no such thing as an amount object. purses and payments hold digital assets of various amounts - could be empty, could be an amount that is non-empty. I think you are thinking of an amount as a thing that a purse or payment has or doesn't have. That's not right. It is a description of assets held. The description could be an empty amount (value = 0 for nat), or non-empty (value = 1 or more for nat).
```suggestion
Each non-empty `purse` and `payment` object contains exactly one
`amount` object. While you may deposit or withdraw an `amount` of digital assets to and from a
```